itests_with_chain/cli_post_test.go

```diff
@@ -0,0 +1,46 @@
+package itests
+
+import (
+	"bytes"
+	"io"
+	"os"
+	"os/exec"
+	"strconv"
+	"testing"
+
+	"golang.org/x/exp/rand"
+)
+
+type CliThing struct {
+	*exec.Cmd
+	*bytes.Buffer
+}
+
+func CliEnv() func(name string, args ...string) *CliThing {
```

Comment:
CURIO_HARMONYDB_HOSTS needs to be set to "yugabyte" to talk to the docker container. Explains the error in failed test.